tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py27,docs
+skip_missing_interpreters = True
```

Comment:
Doesn't this introduce the possibility that tox will report a pass if it has trouble setting up a python 2.7 env? Since there's only one interpreter, should this be false?